packages/protocol/contracts/governance/Election.sol

```diff
@@ -475,14 +501,37 @@ contract Election is
    * @return The total number of votes cast by an account.
    */
   function getTotalVotesByAccount(address account) external view returns (uint256) {
-    uint256 total = 0;
+    bool allowedToVoteOverMaxNumberOfGroupsForAccount = allowedToVoteOverMaxNumberOfGroups[account];
     address[] memory groups = votes.groupsVotedFor[account];
+    require(
+      allowedToVoteOverMaxNumberOfGroupsForAccount || groups.length <= maxNumGroupsVotedFor,
```

Comment:
Is this check necessary as it was already you're already checking this on the setter? https://github.com/celo-org/celo-monorepo/pull/9998/files#diff-0244e44347e8596d09c833006475db2bea054c82b301ede72c7195213ce7e695R1116